backend/tests/apps/ai/management/commands/ai_update_repository_chunks_test.py

```diff
@@ -0,0 +1,125 @@
+from unittest.mock import Mock, patch
+
+import pytest
+
+from apps.ai.management.commands.ai_update_repository_chunks import Command
+
+
+@pytest.fixture
+def command():
+    return Command()
+
+
+@pytest.fixture
+def mock_repository():
+    repository = Mock()
+    repository.id = 1
+    repository.key = "test-repo"
+    repository.is_owasp_site_repository = True
+    repository.is_archived = False
+    repository.is_empty = False
+    return repository
+
+
+class TestAiUpdateRepositoryChunksCommand:
+    def test_command_inheritance(self, command):
+        """Test that the command inherits from BaseChunkCommand."""
+        from apps.ai.common.base.chunk_command import BaseChunkCommand
```

Comment:
I know AI likes doing this but it's you who supposed to ake sure it's done right.
Any reason this import is not a top level one?